torch/cuda/tunable.py

```diff
@@ -240,3 +252,54 @@ def read_file(filename: Optional[str] = None) -> bool:
     if filename is None:
         filename = get_filename()
     return torch._C._cuda_tunableop_read_file(filename)  # type: ignore[attr-defined]
+
+def tune_gemm_in_file(filename: str) -> None:
+    r"""tune GEMM in file."""
+
+    assert is_enabled()
+    assert tuning_is_enabled()
+
+    with open(filename, 'r') as file:
+        for line in file:
+            if line.startswith("Untuned"):
+                untuned_gemm = line.strip().split(',')[1:]
```

Comment:
```suggestion
                untuned_gemm = line.strip().split(',')
```
Based on earlier suggestion of not writing the first column to the file.